### PR TITLE
chore(core): convert notifierService to TS, export from core

### PR DESCRIPTION
**Pull request by anotherchrisberry**
Playing with service workers to install updates on deck-nflx...

### CODE REVIEW

File: app/scripts/modules/core/src/widgets/notifier/notifier.service.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { ISCEService, module } from 'angular';
+import {Subject} from 'rxjs';
```
Comment (jrsquared) [inline app/scripts/modules/core/src/widgets/notifier/notifier.service.ts]: give those brackets some room to breathe!

File: app/scripts/modules/core/src/widgets/notifier/notifier.service.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  private stream = new Subject<INotification>();
+
+  public get messageStream(): Subject<INotification> {
+    return this.stream;
+  }
+
+  constructor(private $sce: ISCEService) { 'ngInject'; }
+
+  public publish(message: INotification): void {
+    console.warn('publishing:', message.body);
```
Comment (jrsquared) [inline app/scripts/modules/core/src/widgets/notifier/notifier.service.ts]: still need the `console.warn`?

**Comment by jrsquared** [comment]
Couple super-minor things; LGTM

**Comment by anotherchrisberry** [comment]
comments addressed (how embarrassing)

**Pull request merged**